web-app/public/locales/en/common.json

```diff
@@ -8,6 +8,11 @@
     "sender": "Sender",
     "receiver": "Receiver",
     "issueDate": "Issue date",
+    "issueDateFrom": "From",
+    "issueDateTo": "To",
+    "uploadDate": "Upload date",
+    "uploadDateFrom": "From",
+    "uploadDateTo": "To",
```

Comment:
you don't have to have `issueDateFrom`, `uploadDateFrom`, `amountFrom`.
Let's unify it